index.js

```diff
@@ -66,6 +67,12 @@ module.exports = {
     }
   },
 
+  validateDependencies() {
+    if ('bootstrap' in this.app.project.bowerDependencies()) {
```

Comment:
But `bootstrap-sass` is a legit dependency (for bs3/sass)? The new blueprint removes Bootstrap from bower.json unconditionally, so we can use this fact to check if the user has run the new blueprint already. If not (bootstrap still in bower), we can warn him.